README.md

```diff
@@ -1,339 +1,103 @@
+So, you if you want to contribute to the Nextcloud's core, you can work in this folder directly !
 
-If your setup isn't working and you can not figure out the reason why, running
-`docker-compose down -v` will remove the relevant containers and volumes,
-allowing you to run `docker-compose up` again from a clean slate.
+Then, you add `nextcloud.local` to your hosts file.
 
-## 🔒 Reverse Proxy
+Finally, the `docker compose up nextcloud proxy` command line. This command line runs the nextcloud, proxy, redis and mailhog containers.
 
-Used for SSL termination. To setup SSL support provide a proper DOMAIN_SUFFIX environment variable and put the certificates to ./data/ssl/ named by the domain name.
+## First connection
 
-You might need to add the domains to your `/etc/hosts` file:
+After running the `docker compose up nextcloud proxy` command, you have to wait a few minutes before trying to connect to your development instance.
 
-```
-127.0.0.1 nextcloud.local
-127.0.0.1 collabora.local
-```
+In fact, you will see that the `nextcloud` and `proxy` containers initialize your Nextcloud, create  user accounts, and so on. Step by step.
 
-This is assuming you have set `DOMAIN_SUFFIX=.local`
+Then, once these steps are completed, you can connect to your development instance. You must enter `http://nextcloud.local` only in your search bar !
 
-You can generate it through:
+In fact, with the `proxy` container you don't need to specify the port number and you can't use `http://localhost` or `http://127.0.0.1`. Just, you have to use this address : `http://nextcloud.local`.
 
-```
-awk -v D=.local '/- [A-z0-9]+\${DOMAIN_SUFFIX}/ {sub("\\$\{DOMAIN_SUFFIX\}", D " 127.0.0.1", $2); print $2}' docker-compose.yml
-```
 
-You can generate selfsigned certificates using:
+### Which user accounts can I use ?
 
-```
-cd data/ssl
-openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout  nextcloud.local.key -out nextcloud.local.crt
-```
+Here is a list of user accounts you can use :
 
-### dnsmasq to resolve wildcard domains
+| uid | password |
+|:---:|:---:|
+| admin | admin |
+| user1 | user1 |
+| user2 | user2 |
+| user3 | user3 |
+| user4 | user4 |
+| user5 | user5 |
+| user6 | user6 |
+| nextcloud | nextcloud |
+| alice | alice |
+| bob | bob |
+| jane | jane |
+| john | john |
 
-Instead of adding the individual container domains to `/etc/hosts` a local dns server like dnsmasq can be used to resolve any domain ending with the configured DOMAIN_SUFFIX in `.env` to localhost.
+## Where can I add my app for development ?
 
-For dnsmasq adding the following configuration would be sufficient for `DOMAIN_SUFFIX=.local`:
-```
-address=/.local/127.0.0.1
-```
-
-### Use valid certificates trusted by your system
-
-* Install mkcert https://github.com/FiloSottile/mkcert
-* Go to `data/ssl`
-* `mkcert nextcloud.local`
-
-* `mv nextcloud.local-key.pem nextcloud.local.key`
-* `mv nextcloud.local.pem nextcloud.local.crt`
-* `docker-compose restart proxy`
```

Comment:
All SSL related info seems gone, whether selfsigned or this version?
Browsers may very well soon drop HTTP support, it’s important to show how to get HTTPS support.

Reply:
No, it didn't disappear.

I move this section to `docs/containers/ssl.md`. But, I forgot to add a link here : https://github.com/arawa/nextcloud-docker-dev/blob/13a001965c2e7472474295aa361fdd26baa0a204/README.md?plain=1#L89 ^^'

I am going to fix it ^^'

Reply:
I fixed it here : https://github.com/juliushaertl/nextcloud-docker-dev/commit/b501b22a3b9d9b23baf20c0d523d460bac789d44

It's okay for you, or do you feel that something needs to be changed ? Don't hesitate to tell me :)